lib/checks/aria/aria-errormessage-evaluate.js

```diff
@@ -45,14 +45,13 @@ function ariaErrormessageEvaluate(node, options) {
 					-1
```

Comment:
I think this is most likely a separate issue (I didn't make any change because I felt it was out of scope, and I wasn't 100% sure about it), but right now it looks like the `ariaErrormessageEvaluate` check will fail in a valid case such as this (from https://www.w3.org/TR/2017/REC-wai-aria-1.1-20171214/#aria-errormessage)
```html
<label for="startTime"> Please enter a start time for the meeting: </label>
<input id="startTime" type="text" aria-errormessage="msgID" value="" aria-invalid="false">
<span id="msgID" aria-live="off" style="visibility:hidden"> Invalid time:  the time must be between 9:00 AM and 5:00 PM" </span>
```
where `aria-live` is `off` because `aria-invalid` is `false`. Maybe this check should check for the existence of `aria-invalid="true"` before getting strict about announcers?

Sidenote, w3 also seems to suggest that the announcer is optional on the element referenced by `aria-errormessage`:
> Authors MAY use live regions for the error message element applying either an aria-live property or using one of the live region roles, for example, alert. A live region scenario is when an error message is displayed to users only after they have provided invalid information. 

Just curious about your thoughts regarding this 🙂

Reply:
That's a totally valid point. Would you mind opening an issue for this?

Reply:
Absolutely! I wouldn't mind working on it as well.